docs/data.mdx

```diff
@@ -52,6 +52,37 @@ Any small enough tabular data files, such as tsv, csv, BED, BEDPE, and GFF, can
 
 <TableWrapper objName='CsvData' GoslingSchema={GoslingSchema} />
 
+### GFF3 (No HiGlass Server)
+
+This format allows for files that follow the [GFF3 specification](https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md). 
+
+Currently, the GFF3 file must have an accompanying index file. If you do not have an index file for your GFF3 file, you 
+can create one using [tabix](http://www.htslib.org/doc/tabix.html). Otherwise, you can treat the GFF3 file as if it were a 
+CSV file and use the CSV data specification, but this will not be as performant for large files. 
+
+The field names correspond to the names of the columns. For example, the field which corresponds to the "start" column,
+is called "start". 
+
+[GFF file demo](https://gosling-lang.github.io/gosling.js/?example=doc_gff)
+
+```javascript
+{
+  "tracks":[{
+    "data": {
+      "url": "https://s3.amazonaws.com/gosling-lang.org/data/gff/Homo_sapiens.GRCh38.94.chr.gff3.gz",
+      "indexUrl": "https://s3.amazonaws.com/gosling-lang.org/data/gff/Homo_sapiens.GRCh38.94.chr.gff3.gz.tbi",
+      "type": "gff",
+      "extractAttributes": true
+    },
+    "mark": "rect", 
+      "x": {"field": "start"}, // example using one of the standard fields
+      "xe": {"field": "end"},
+    ... // other configurations of this track
+  }]
+}
+```
+
```

Comment:
@etowahadams Thanks for adding this! I found your GFF examples very helpful for understanding how data are parsed (especially for child attributes). 

https://github.com/gosling-lang/gosling.js/pull/923#issuecomment-1624510076

Do you think it's a good idea to add such an example to the documentation (e.g., what the original and parsed data look like with and without the `extractAttributes` option)?

Reply:
Added a similar example!

Reply:
Thanks!